packages/scandipwa/src/component/ProductInformation/ProductInformation.style.scss

```diff
@@ -10,6 +10,18 @@
  * @link https://github.com/scandipwa/base-theme
  */
 
+ @mixin alltags {
```

Comment:
Read above comment, place mixin in right way

Reply:
And maybe think about changing name for mixin ???

Reply:
will do ty